src/content/peerconnection/audio/js/main.js

```diff
@@ -241,6 +248,20 @@ function setDefaultCodec(mLine, payload) {
   return newLine.join(' ');
 }
 
+// Change the ptime. For opus supported values are [10, 20, 40, 60].
+// eslint-disable-next-line no-unused-vars
```

Comment:
can you add "For experts at the console only" here too? And add ptime to the description?

Reply:
done. moved to the top too to increase visibility.